plugin.go

```diff
@@ -26,6 +65,14 @@ func (p ConfigPlugin) Setup(c *Config) error {
 	}
 }
 
+func (p ConfigPlugin) ModifyFuncMap(funcMap template.FuncMap) template.FuncMap {
+	modified := make(template.FuncMap, len(funcMap))
+	for funcName, f := range funcMap {
+		modified[strings.ToLower(p.FuncPrefix)+funcName] = f
```

Comment:
`strings.ToLower` seems unnecessary because template function names need not necessarily be in lowercase.